hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/operations/CustomCreateOperation.java

```diff
@@ -50,7 +51,7 @@ protected long cost(@NonNull final MessageFrame frame) {
         final var updater = (ProxyWorldUpdater) frame.getWorldUpdater();
```

Comment:
As long as you're adding checks, doesn't the argument `frame` need one?  (If it does, same goes for `CustomCreate2Operation.setupPendingCreation`.)

(I suppose you could argue that if `frame` was ever `null` here in the contract service, at least by the time you got down here, something is so seriously wrong that in fact you could never have got here in the first place! I don't see any other nullity checks for `frame` so that must be what's going on.)

Reply:
Yes agreed if frame is null then something catastrophic has happened.